src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,87 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (pageNumber: number, itemsPerPage: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePageChange = (pageNumber: number) => {
+    onPageChange(pageNumber, perPage);
+  };
+
+  const renderPageNumbers = () => {
+    const pageNumbers = [];
+
+    // eslint-disable-next-line no-plusplus
+    for (let i = 1; i <= totalPages; i++) {
```

Comment:
const pages = new Array(totalPages).fill(null).map((_, index) => index + 1);

and than u may use this variable by map